.github/workflows/pythonpackage.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.8]
```

Comment:
not sure we want to drop support for python3.7. at the moment all docker images are using python 3.7

Reply:
makes sense, coming back to the previous setup